src/Data/Rational/Unnormalised.agda

```diff
@@ -38,7 +41,7 @@ record ℚᵘ : Set where
   denominatorℕ = suc denominator-1
 
   denominator : ℤ
-  denominator = + denominatorℕ
+  denominator = ℤ.+ denominatorℕ
```

Comment:
Is there a reason you've needed to add `ℤ.` to all the `+_`?

Reply:
Can't remember, I think there used to be conflicts but they're gone so I removed the `ℤ.` again.